### PR TITLE
adding ctime to sched-process-exec event

### PR DESCRIPTION
**Pull request by mtcherni95**

### CODE REVIEW

**Comment by mtcherni95** [comment]
solves #1075

**Comment by mtcherni95** [comment]
> We recently added ctime in #1041. This approach, however, is better as it is not exposed to race conditions. So, I suggest removing the ctime calculation made by "exec-info" flag, and change the name of this flag to exec-hash or something similar

alright, sounds good to me.

**Comment by mtcherni95** [comment]
If no other comment/review, then we can promote IMO @itaysk @yanivagman

File: tracee-ebpf/tracee/events_processor.go [hunk 1]
```diff
@@ -197,11 +197,6 @@ func (t *Tracee) processEvent(ctx *context, args map[string]interface{}, argMeta
 					*argMetas = append(*argMetas, hashMeta)
 					ctx.Argnum += 1
 					args["sha256"] = currentHash
-
-					ctimeMeta := external.ArgMeta{"ctime", "unsigned long"}
-					*argMetas = append(*argMetas, ctimeMeta)
-					ctx.Argnum += 1
-					args["ctime"] = sourceFileCtime
```
Comment (yanivagman) [inline tracee-ebpf/tracee/events_processor.go:204]: With this change in place, there is no more reason to have sourceFileCtime and even sourceFileStat is not required anymore, right? If so, can we just remove those and replace it by the ctime argument of the sched_process_exec event?

**Review by yanivagman — APPROVED** [review]

**Pull request merged**